### DISCUSSION TITLE
Keda as custom/external metrics provider

### DISCUSSION THREAD

**Discussion by grzesuav** [kedacore/keda#2762] [Q&A / Need Help] [answered]
HI, it is unclear for me if I can use KEAD just as a custom and external metrics provider, so that whole autoscaling will  be done by HPA. Would it be possible? Is there any documentation on that ?

Best,

Grzegorz

**Comment by zroubalik** [comment, marked as answer]
That's the way how KEDA works currently. In fact it provides metrics for HPA -> that drives the scaling  1<->N replicas and KEDA does the 0<->1 part.

**Reply by grzesuav** [reply]
thanks. Actually, do  scaling 0 <-> 1 (done by keda) can be disabled ?

**Reply by zroubalik** [reply]
just set `minReplicaCount` to  1
